docs/code/getting-started.md

```diff
@@ -675,4 +675,123 @@ Let’s recap what we learned in this section:
 
+#Notifications {#notifications}
+If you have been using elementary os, by now you've probably already seen the white bubbles that appear on the top right of the screen. Those are notifications! Notifications are simple ways to notify a user about the state of your app. They can inform the user that a long process has been completed, or a new message has arrived. Either way, in this section we are going to show you just how to get them to work in your app. Let's begin by making a new project!
+
+##Making Preparations {#making-preparations}
+1. Create a new folder inside of  "~/Projects" called "notifications-app"
+2. Create a file inside called ```notify-app.vala ```
+3. Re create the CMake folder and CMakeFiles.txt file If you don't remember how to, i recommend you go back to the [previous section](#building-and-installing-with-cmake). But you can also use the files from your previous projects, and modify it accordingly.
+4. Remember how to [make a .desktop file](#the-desktop-file)? Excellent! Well, make one once again for this project. But since your app will be displaying applications, add `X-GNOME-UsesNotifications=true` to the end of the file. This is needed so your app's notification settings can automatically be handled by switchboard's notifications plug.
+
+### Gtk.Application {#gtk-application}
+Sadly, we can't just add a new notification right away. You are going to need your program to extend from the class Gtk.Application. Gtk.Application is a class that handles many important aspects of a Gtk app, it will also allow you to give your app some new features, one of them being notifications! If you want some more details about what else it can do, you can always check out [valadoc](http://valadoc.org/#!api=gtk+-3.0/Gtk.Application)
+
+##Writing your app! {#writing-your-app}
+Now that you know what a Gtk.Application is, it's time to begin coding your app! Let's start with setting up the basic layout:
+
+	public class MyApp : Gtk.Application {
+
+		public MyApp () {
+			Object (application_id: "my.app",
+			flags: ApplicationFlags.FLAGS_NONE);
+		}
+    
+		protected override void activate () {
+			var app_window = new Gtk.ApplicationWindow (this);
+    
+			app_window.show ();
+		}
+    
+		public static int main (string[] args) {
+			MyApp app = new MyApp ();
+			return app.run (args);
+		}
+	}
+    
+You will notice though that the way you initiate your app will look a little different from what we did a few sections back. This time, in your `main` you are starting your app with `app.run` and you have a new function called `activate` inside of your class; This `activate` function will be the one that executes when you invoke `app.run`. We are also creating a Application-Window, this is where you will place all the widgets your app needs. Now that we have a simple window, let's use what we learned in [creating layouts](#gtk-grid) and make a grid containing two buttons, one that will show a notification, and one that will hide it! 
```

Comment:
- You will notice though that the way you initiate your app will look a little different from what we did a few sections back
+ Initiating your app with `Gtk.Application` is a little different from what we did a few sections back.